packages/apps/clickhouse/templates/clickhouse.yaml

```diff
@@ -3,16 +3,27 @@ kind: "ClickHouseInstallation"
 metadata:
   name: "{{ .Release.Name }}"
 spec:
+  {{- with .Values.size }}
   defaults:
     templates:
       dataVolumeClaimTemplate: data-volume-template
-      logVolumeClaimTemplate: log-volume-template
+  {{- end }}
   configuration:
+    {{- with .Values.users }}
```

Comment:
what will happen if we are removing user from `values.yaml`?

Reply:
The whole block will be omited

Reply:
I don't mean on Helm level, but what will happen on application level.

Reply:
I am not talking about INITIALIZE phase, but rather - about lifecycle. So when the cluster already created and you are changing the values.

Reply:
ClickHouse will recognize the file automatically without restarting and you can login using a command like the following:

https://altinity.com/blog/goodbye-xml-hello-sql-clickhouse-user-management-goes-pro